core/examples/http_bench.js

```diff
@@ -85,20 +85,20 @@ function listen() {
 
 /** Accepts a connection, returns rid. */
 async function accept(rid) {
-  return await sendAsync(ops["accept"], rid);
+  return sendAsync(ops["accept"], rid);
 }
 
 /**
  * Reads a packet from the rid, presumably an http request. data is ignored.
  * Returns bytes read.
  */
 async function read(rid, data) {
-  return await sendAsync(ops["read"], rid, data);
+  return sendAsync(ops["read"], rid, data);
 }
 
 /** Writes a fixed HTTP response to the socket rid. Returns bytes written. */
 async function write(rid, data) {
-  return await sendAsync(ops["write"], rid, data);
+  return sendAsync(ops["write"], rid, data);
```

Comment:
It will be interesting to see if this has any effect.

Reply:
It should.  It is at least one less microtask, unless V8 optimises it out, which it might be smart enough to do.